### ISSUE TITLE
not going into timeout case

### ISSUE THREAD

**Issue by surendiranganesan** [linslin/Yii2-Curl#36]
Hi,

I have tried the example shown in your main page.

```
public function actionGetAdvancedExample()
{
        //Init curl
        $curl = new curl\Curl();

        //get http://example.com/
        $response = $curl->post('http://www.google.com:81/');

        // List of status codes here http://en.wikipedia.org/wiki/List_of_HTTP_status_codes
        switch ($curl->responseCode) {

            case 'timeout':
                //timeout error logic here
                break;

            case 200:
                //success logic here
                break;

            case 404:
                //404 Error logic here
                break;
        }
}
```

but its not going into timeout case its going into default case only.

kindly help me on this i tried with both get and post.

regards,

**Comment by linslin** [comment]
Hi @surendiranganesan,

many thanks for your response. We gonna check this the next days.

**Comment by linslin** [comment]
Thanks, its fixed in 1.0.9.
